### PR TITLE
Add Int8Range and NumRange

### PR DESCRIPTION
**Pull request by PJUllrich**

### CODE REVIEW

File: test/int8range_test.exs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+defmodule Ecto.Int8RangeTest do
+  use Ecto.DateRange.DataCase
```
Comment (davydog187) [inline test/int8range_test.exs:2]: lets add a doc test so we catch the bug in the docs

```suggestion
  use Ecto.DateRange.DataCase
  
  doctest Ecto.Int8Range
```

File: test/numrange_test.exs [hunk 2]
```diff
@@ -0,0 +1,67 @@
+defmodule Ecto.NumRangeTest do
+  use Ecto.DateRange.DataCase
```
Comment (davydog187) [inline test/numrange_test.exs:2]: same, doc test

**Comment by PJUllrich** [comment]
Good idea 🎉 will commit and test it tomorrow

**Review by davydog187 — APPROVED** [review]
🚀🚀🚀

**Pull request merged**